tests/AuthorizedAccessTokenControllerTest.php

```diff
@@ -73,6 +83,8 @@ public function test_tokens_can_be_deleted()
     {
         $request = Request::create('/', 'GET');
 
+        $this->events->shouldReceive('dispatch')->once();
```

Comment:
It's probably better to assert the actual `AccessTokenRevoked` was dispatched.

Reply:
Ready.